src/kbmod/search/KBMOSearch.h

```diff
@@ -36,26 +36,26 @@ class KBMOSearch {
     void setDebug(bool d);
 
     // The primary search functions.
-    void enableGPUSigmaGFilter(std::vector<float> pyPercentiles, float pySigmaGCoeff, float pyMinLH);
+    void enableGPUSigmaGFilter(std::vector<float> py_percentiles, float pysigmag_coeff, float pymin_lh);
     void enableCorr(std::vector<float> pyBaryCorrCoeff);
```

Comment:
`pyBarryCoeff`, you'll probably catch it when doing the above one. Also, `float pymin_lh`, `float `py_min_lh` or `float min_lh`